_data/sites.json

```diff
@@ -6414,12 +6414,8 @@
         "meta": "popular",
         "name": "Reddit",
         "url": "https://ssl.reddit.com/prefs/delete/",
-        "difficulty": "easy",
-        "notes": "Increase your productivity by over 5 times with this one trick they don't want you to know about!",
-        "notes_it": "Incrementa la tua produttività di oltre 5 volte con questi consigli che loro non vogliono farti sapere!",
-        "notes_pt_br": "Aumente sua produtividade em 5 vezes com este truque que eles não querem que você saiba!",
-        "notes_cat": "Augmenti la seva productivitat fins a 5 vegades amb aquest truc que ningú en vol sentir a parlar!",
-        "notes_es": "Aumente tu productividad hasta 5 veces con este truco del que nadie quiere oír hablar!",
+        "difficulty": "hard",
+        "notes": "Reddit does not allow for the actual deletion of your account but rather just let's you deactivate it. Your content is also not deleted automatically: \"Deactivating your account will not delete the content of posts and comments you've made on reddit. To do so, please delete them individually.\". If you are an EU citizen, you may request an actual deletion of your personal data: \"To contact us about exercising data subject rights under GDPR, please write to eurepresentative@reddit.com\".",
```

Comment:
This is a good update, however is rather wordy. What do you think of the following rewording of it?
```suggestion
        "notes": "You cannot delete your Reddit account; you can only deactivate it. Content posted to Reddit (posts and comments) must be deleted individually. If you are an EU citizen, <a href=\"mailto:eurepresentative@reddit.com\">you may request deletion of your personal data via email</a>.",
```